### PR TITLE
[infra] Migrate GitHub Actions to Travis

### PR DESCRIPTION
**Pull request by jaylees14**
- Migrate GitHub Actions to Travis
- Break each step into a named travis stage
- Make sure we're actually formatting test & integration test files

NB: This currently runs every stage on every commit - wondering if I should just run the tests on commit then IT/Coverage/Format on PR and merges?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+language: scala
+
+scala:
+    - 2.13.1
```
Comment (c-harding) [inline .travis.yml]: The four spaces do not look like yaml to me

Comment (c-harding) [reply .travis.yml]: And doesn’t this work without being an array?

Comment (jaylees14) [reply .travis.yml]: https://docs.travis-ci.com/user/languages/scala/#specifying-scala-versions

**Review by c-harding — APPROVED** [review]
I trust it runs the tests and fails successfully when they don’t pass?

**Review by Lewiky — APPROVED** [review]
LGTM

**Review by willjvsmith — APPROVED** [review]

**Pull request merged**